index.html

```diff
@@ -2284,17 +2313,23 @@
         presentation.
       </p>
       <p>
-        This specification does not prohibit a user agent from publishing
-        information about its <a>set of presentations</a>. The group envisions
-        a user agent on another device (distinct from the controller or
-        presentation) becoming authorized to reconnect to the presentation,
-        either by user action or by discovering the presentation's URL and id.
+        This specification allows a user agent to publish information about its
+        <a>set of presentations</a>, and allows a browsing context on another
+        user agent connect to a running presentation via <code><a for=
+        "PresentationRequest">reconnect</a>()</code>. To connect, the
+        additional browsing context must discover the presentation URL and
+        presentation ID of the presentation, either provided by the user, or
+        via a shared service.
+      </p>
+      <p>
+        However, the specification makes makes no guarantee as to the identity
+        of the connecting party. Once connected, the <a>receiving browsing
+        context</a> may wish to further verify its identity through
+        application-specific means. For example, the connecting <a>controlling
+        browsing context</a> may provide a token via <code><a for=
+        "PresentationConnection">send</a>()</code> that the <a>receiving
+        browsing context</a> can verify corresponds to a authorized entity.
```

Comment:
Good point.  Reworded this section as suggested, and fixed the other grammatical errors noted.